### DISCUSSION TITLE
Friendly image/file URL

### DISCUSSION THREAD

**Discussion by CallMeLaNN** [directus/directus#10575] [Feature Requests]
I think currently only `/assets/<uuid>` only the possible way to get an image, right?

There are discussions about like https://github.com/directus/directus/discussions/9079 https://github.com/directus/directus/discussions/7612 However I'm not referring to that but just a virtual path to the file.

When I create a content management, taking care of SEO stuff, creating meta, navigations, slugs etc is ok in a collection but user can't control the image URL or slug. Other than SEO and user friendly name, file extension also give some good hint for user and some client that the link is an image.

If I create an endpoint, I will double stream the file.

I'm thinking to find a way for user to keep the filename and folder as it is. e.g. `/assets/parent/child/file.jpg` or any name like having a slug field or reusing `Filename` field. I'm not sure about this should include on the fly transformation and preset but I believe having a friendly name is useful.

**Comment by w0kyj** [comment]
I think this is supported with #13907

**Reply by rijkvanzanten** [reply]
It is!
